sqlglot/dialects.py

```diff
@@ -695,6 +695,8 @@ def _create_sql(self, e):
 
         if kind.upper() == "TABLE" and temporary is True:
             return f"CREATE TEMPORARY VIEW {self.sql(e, 'this')} AS {self.sql(e, 'expression')}"
+        if kind.upper() == "FUNCTION" and temporary is True:
```

Comment:
why is this necessary? the reason why the previos line exists is because table != view but function should work?

Reply:
You are right